include/seqan3/range/views/repeat.hpp

```diff
@@ -205,14 +205,14 @@ class repeat_view : public std::ranges::view_interface<repeat_view<value_t>>
 
 private:
     //!\brief A std::views::single over the input.
-    decltype(std::views::single(std::declval<value_t &>())) single_value;
-}; //class repeat_view
```

Comment:
Why did you delete this comment? Isn't it helpful to now what this bracket closes?

Reply:
It isn't in our style guide to do an end comment for classes. We do this only for namespaces and macros. I removed this for consistency.